### PR TITLE
Add more content to the Completions guide

### PR DESCRIPTION
**Pull request by yixu34**
https://app.shortcut.com/scaleai/story/782493/guide-update-completions-guide

Roughly tried to follow https://platform.openai.com/docs/guides/gpt/completions-api

image
image
image

### CODE REVIEW

**Comment by yixu34** [comment]
TBH this recycles some info from the API reference, but I think that's OK. This is presenting things in a gentler, high-level format, before slamming the user with tables.

File: docs/guides/completions.md [hunk 1]
```diff
@@ -1,3 +1,83 @@
-Given a prompt, the model will return one or more predicted Completions. The Completions API can be run either
-synchronous or asynchronously (via Python `asyncio`); for each of these modes, you can also choose to stream token
-responses or not.
+LLM Engine provides a list of open source language models (see [Model Zoo](/model_zoo)) that can be used for producing
```
Comment (seanshi-scale) [inline docs/guides/completions.md:1]: nit: "provides access to various open source..."? feel like the word "list" is unnecessary

Comment (seanshi-scale) [reply docs/guides/completions.md:1]: probably can be worded differently though

File: docs/guides/completions.md [hunk 2]
```diff
@@ -1,3 +1,83 @@
...
+    max_new_tokens=10,
+    temperature=0.2,
+)
+```
+
+See the full [API reference documentation](/api/python_client/#llmengine.Completion) to learn more.
+
+## Completions response format
+
+An example completions API response looks as follows:
```
Comment (seanshi-scale) [inline docs/guides/completions.md:18]: should "completions" be capitalized?

**Pull request merged**